spec/SensioLabs/Behat/PageObjectExtension/PageObject/Factory/CamelcasedClassNameResolverSpec.php

```diff
@@ -77,4 +86,22 @@ function it_throws_an_exception_if_element_class_does_not_exist()
         $this->shouldThrow(new \InvalidArgumentException($expectedMessage))
             ->duringResolveElement('Recent searches');
     }
+
+    function it_resolves_an_element_class_given_as_fqcn()
+    {
+        $this->beConstructedWith(array('\\Foo', '\\Element'));
+
+        $elementFQCN = 'spec\\SensioLabs\\Behat\\PageObjectExtension\\PageObject\\Factory\\Bar';
+        $this->resolvePage($elementFQCN)->shouldReturn($elementFQCN);
+    }
+}
+
+class Foo extends Page
```

Comment:
Do you mean arguments passed to the class under test? These are only namespace prefixes. In your case these would be ignored, as a FQCN is passed to the resolve method.